worker/meterstatus/isolated.go

```diff
@@ -9,10 +9,10 @@ import (
 	"github.com/juju/errors"
 	"github.com/juju/utils/clock"
 	"gopkg.in/juju/charm.v6/hooks"
-	worker "gopkg.in/juju/worker.v1"
+	"gopkg.in/juju/worker.v1"
 	"gopkg.in/tomb.v1"
 
-	"github.com/juju/juju/worker/uniter/runner/context"
+	"github.com/juju/juju/worker/common/runner"
```

Comment:
"runner" is far too generic without the context of uniter in the path. Maybe "charmrunner"?

Reply:
done